### PR TITLE
PlayerJS: Refactor with access to widget for each canvas

### PR DESCRIPTION
**Pull request by rubenpingol-xibo**
relates to xibosignageltd/xibo-private#567

### CODE REVIEW

**Comment by rubenpingol-xibo** [comment]
TEST: /Layout END

File: modules/src/player.js [hunk 1]
```diff
@@ -19,799 +19,5 @@
...
-              widget.data,
-              widget.showError,
-              widget.errorMessage,
-              widget.data,
-            );
-          }
-        });
-      }
-    });
+  XiboPlayer.init(widgetData, elements);
```
Comment (maurofmferrao) [inline modules/src/player.js:22]: Do we still need this file? Or could we just use `xibo-player.js` to initialize instead?

**Review by maurofmferrao — COMMENTED** [review]
Everything looks good to me, I just have a small question about the structure :)

File: modules/src/player.js (hunk 1 above)
Comment (rubenpingol-xibo) [reply modules/src/player.js:22]: I've thought about this as well 😅 Should we just add it after the `module.exports` on `xibo-player.js`?

**Comment by rubenpingol-xibo** [comment]
TEST: /Layout END

**Review by maurofmferrao — APPROVED** [review]
Looks good to me!

**Pull request merged**